platforms/web/index.html

```diff
@@ -3,7 +3,7 @@
     <head>
         <meta charset="UTF-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <title>Wysisyg</title>
+        <title>Wysiwyg</title>
```

Comment:
This had been winding me up for a while and needed addressing before it became critical for me.

Reply:
```suggestion
        <title>Rich Text Editor</title>
```
Wysiwyg is also not on fleek.
Fixed it for you. 😂